packages/xo-server/config.json

```diff
@@ -46,7 +46,7 @@
     "mountsDir": "/run/xo-server/mounts",
 
     // timeout in milliseconds (set to 0 to disable)
-    "timeout": 600e3,
+    "timeout": 600e3
```

Comment:
Did it cause any issues?

Reply:
No. Are you talking about the comma? I think it's because of the rebase and the formatting.

Reply:
Ok, never mind this then, I'll change this file from JSON to TOML anyway because I find JSON very unpractical for config (no real comments support, no trailing commas, etc).

Reply:
I agree.